Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -98,6 +98,7 @@ void ABenchmarkGymGameMode::Tick(float DeltaSeconds)
 			SecondsTillPlayerCheck -= DeltaSeconds;
 			if (SecondsTillPlayerCheck <= 0.0f && GetNumPlayers() != ExpectedPlayers)
 			{
+				// This log is used by the NFR pipeline to indicate if a client failed to connect
 				UE_LOG(LogBenchmarkGym, Error, TEXT("A client connection was dropped. Expected %d, got %d"), ExpectedPlayers, GetNumPlayers());
```

Comment:
should we output a success message here if we're good?

Reply:
Sounds good to me, pushed